### DISCUSSION TITLE
Table: Mark as measure table

### DISCUSSION THREAD

**Discussion by MarioMue13** [TabularEditor/TabularEditor3#1112] [Ideas]
If you follow the approach of putting all measures to one dummy table, it would be helpful to pin this table to the top of all tables

**Comment by otykier** [comment]
Just to be clear, you’re asking for the default sort order in the TOM Explorer to show tables that contain measures but no visible columns, at the top of the list, similar to PBI Desktop?

**Comment by MarioMue13** [comment]
Hi Daniel, yes exactly, this would be a generic description. In our specific approach we have one FactMeasures (dummy table) that manages all measures. So I thought of a "marking" option

**Comment by mlonsk** [comment]
Hi @MarioMue13 
Does the release of [Table Groups](https://docs.tabulareditor.com/te3/features/table-groups.html) solve this feature request for you?

**Comment by MarioMue13** [comment]
Hi Morten,

Thanks for reaching out. Yes definitely :). Keep going guy ;)

Am Mi., 1. Nov. 2023 um 20:19 Uhr schrieb Morten Lønskov <
***@***.***>:
